hub/values.yaml

```diff
@@ -222,6 +222,13 @@ jupyterhub:
       tag: '0.0.1-0.dev.git.7126.h64800628'
     networkPolicy:
       enabled: true
+      # interNamespaceAccessLabels=accept makes the hub pod's associated
+      # NetworkPolicy accept ingress from pods in other namespaces that has a
+      # hub.jupyter.org/network-access-hub=true label.
+      #
+      # ref: https://z2jh.jupyter.org/en/stable/resources/reference.html#hub-networkpolicy-internamespaceaccesslabels
+      #
+      interNamespaceAccessLabels: accept
       ingress:
         - from:
             - namespaceSelector:
```

Comment:
also perhaps we don't need the following block any more?  see https://github.com/2i2c-org/infrastructure/pull/2648/files#diff-301db94aa87a33a484be445796919615b3eecdc883900e976be6870e645f6d48L533